---
bumperbot_detection/LitterPoint next_litter
bool success
